packages/app/src/debug/DebugSpec.vue

```diff
@@ -16,11 +16,64 @@
           class="flex w-full grid px-18px gap-y-8px items-center"
         >
           <div class="flex-grow flex w-full gap-x-2 truncate items-center">
-            <IconDocumentText
-              stroke-color="gray-500"
-              fill-color="gray-100"
-              size="16"
-            />
+            <Tooltip
+              v-if="foundLocally"
+              placement="bottom"
+              color="dark"
+              data-cy="open-in-ide"
+            >
+              <OpenFileInIDE
+                v-slot="{onClick}"
+                :file-path="specData.fullPath"
+              >
+                <button
+                  class="rounded-md border-1px border-gray-100 p-4px group hocus:border-indigo-200"
+                  @click="onClick"
+                >
+                  <IconDocumentText
+                    stroke-color="gray-500"
+                    fill-color="gray-100"
+                    hocus-stroke-color="indigo-400"
+                    hocus-fill-color="indigo-200"
+                    size="16"
+                    interactive-colors-on-group
+                  />
+                </button>
+              </OpenFileInIDE>
+              <template
+                #popper
+              >
+                <div
+                  class="text-center text-sm max-w-240px"
+                  data-cy="open-in-ide-tooltip"
+                >
+                  {{ t('debugPage.openFile.openInIDE') }}
+                </div>
+              </template>
+            </Tooltip>
+            <Tooltip
+              v-else
+              class="rounded-md border-1px border-gray-100 p-4px"
+              placement="bottom"
+              color="dark"
+              data-cy="open-in-ide-disabled"
+            >
```

Comment:
Tooltip is rendering right under the icon, instead of outside the border. I think you need to apply the order to a `div` surrounding the icon and then wrap the tooltip around that.

Also, not sure why the SVG is getting a tab index.  When using keyboard controls, the focus stops on this wrapped SVG, but I cannot figure out why.  Since there is no button in the state where the spec is not found locally, it should not gain focus from keyboard controls.  Any ideas how to prevent this?

<img width="378" alt="Screenshot 2023-02-03 at 10 26 25 AM" src="https://user-images.githubusercontent.com/1702361/216643289-cdcc02c4-adf5-49bb-bffc-2dbd16ccbe3d.png">

Reply:
I assume it's bound to the tooltip component. Even if it is not clickable, this behavior makes sense to me. You'd want to be able to tab into and see the tooltip, even if you can't interact with it

Reply:
Addressed in [dab6639](https://github.com/cypress-io/cypress/pull/25691/commits/dab6639dbf5507643dc69f84109964232cab2229), converted this to an `aria-disabled` button so it maintains focusability but the focus is around the entire container rather than just the svg